authcrypt/main.cpp

```diff
@@ -24,8 +24,13 @@
 #include "mbedtls/platform.h"
 
 int main() {
+    mbedtls_platform_context platform_ctx;
+    if(mbedtls_platform_setup(&platform_ctx) != 0) {
+        return -1;
+    }
+
     int exit_code = MBEDTLS_EXIT_SUCCESS;
-    Authcrypt *authcrypt = new Authcrypt();
+    Authcrypt *authcrypt = new Authcrypt(&platform_ctx);
```

Comment:
Question: I read the conversation about the location of these calls and the context. My understanding is that @RonEld suggested putting them closer to the crypto stuff. In the case of this particular example (and probably tls-client), why not make the struct part of the class itself?

Reply:
The platform initialization itself is not connected to a particular test starting. It's an external dependency which has to be satisfied. If someone was to split a test in two, or add another one, there should be only one platform initialization done independently of the test initialization.